### PR TITLE
question description and code added

### PR DESCRIPTION
**Pull request by baazis**

### CODE REVIEW

File: 2021/september/day-1/array_nesting.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Problem Description
```
Comment (YashKumarVerma) [inline 2021/september/day-1/array_nesting.md:1]: remove problem description heading

File: 2021/september/day-1/array_nesting.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+One of the longest sets s[k]:
+
+s[0] = {nums[0], nums[5], nums[6], nums[2]} = {5, 6, 2, 0}
+
+Example 2:
+
+Input: nums = [0,1,2]
+
+Output: 1
```
Comment (YashKumarVerma) [inline 2021/september/day-1/array_nesting.md:36]: wrap this in a code block
```
like this
and this
```

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Comment by YashKumarVerma** [comment]
1. change PR title to "<month> : <day #> : <language>"
2. Try squashing commits to one. read about git rebase

**Comment by baazis** [comment]
done changes

**Pull request closed without merging**